frontend/admin/src/js/components/classification/UpdateClassification.tsx

```diff
@@ -204,13 +208,16 @@ export const UpdateClassificationForm: React.FunctionComponent<
   );
 };
 
-const UpdateClassification: React.FunctionComponent<{
-  classificationId: string;
-}> = ({ classificationId }) => {
+type RouteParams = {
+  classificationId: Scalars["ID"];
+};
+
+const UpdateClassification = () => {
   const intl = useIntl();
+  const { classificationId } = useParams<RouteParams>();
   const [{ data: classificationData, fetching, error }] =
     useGetClassificationQuery({
-      variables: { id: classificationId },
+      variables: { id: classificationId || "" },
     });
```

Comment:
we should prob make a ticket to start throwing errors if this param as ever null, since this suppresses the warning, but it should never happen as far as I can tell, right?

Reply:
Agreed! 💯

Reply:
Added to spin-out ticket list below!